README.md

```diff
@@ -19,11 +19,11 @@ You have to activate the bundle:
 ```php
 // in app/AppKernel.php
 public function registerBundles() {
```

Comment:
According to the [documentation](https://symfony.com/doc/current/bundles/best_practices.html#installation-instructions), it's a best practice to describe instructions for both flex and legacy ways in README.

Reply:
Ah nice catch, I didn't know this page